binary_transparency/firmware/internal/verify/bundle.go

```diff
@@ -74,3 +80,14 @@ func verifyBundle(b api.ProofBundle) (api.FirmwareMetadata, error) {
 	}
 	return fwMeta, nil
 }
+
+// BundleForConsistency checks for consistency between Bundle checkpoint,
+// and the checkpoint stored on the device
+func BundleForConsistency(bc api.LogCheckpoint, dc api.LogCheckpoint, cp [][]byte) error {
```

Comment:
Could you move the functionality here into `BundleForUpdate`?

The exported methods in this package are named for their use case, so `verify.BundleForUpdate()` would check everything is correct before the caller would proceed to install the update, similarly `verify.BundleForBoot()` would perform checks applicable at boot time.

The idea is that having just two entry points in here makes it harder for accidents to happen with each newly supported device being added.

Reply:
Lets discuss! I agree with the philosophy behind the concept. However, in terms of practicality, consistency proof, requires a lot more information making the function arguments complex.